projects/packages/blaze/src/class-dashboard-config-data.php

```diff
@@ -39,6 +40,7 @@ public function get_data() {
 			'admin_page_base'          => $this->get_admin_path(),
 			'api_root'                 => esc_url_raw( rest_url() ),
 			'blog_id'                  => $blog_id,
+			'jetpack_version'          => Constants::get_constant( 'JETPACK__VERSION' ),
```

Comment:
If you move this option below, were we have all of the site information resides, you won't need to add the additional selector in https://github.com/Automattic/wp-calypso/pull/83590/files.

When jetpack Blaze loads, it loads the store using the `intial_state` prop. It will take the correct `jetpack_version` if you put it under `sites->items->[blog_id]`

It will make the Blaze loading easier, and we won't need to add another selector to Calypso.
What do you think?

Reply:
Indeed, I didn't consider that option. Will do so, thanks @sbarbosa.

Reply:
Moved in e4d9b952c6904c67eb7f113b9c9220801e4ffe30.